resources/js/components/ProfileSideBar.tsx

```diff
@@ -4,7 +4,7 @@ import { FaRegCircleUser } from 'react-icons/fa6';
 import { TiEdit } from 'react-icons/ti';
 import { FiArchive } from 'react-icons/fi';
 import User from '../models/User';
-import avatar from '../../../public/assets/images/avatar.png';
```

Comment:
as I have mentioned it before including images to your bundle is not a good idea,  but in this case for such basic and small image it might be okey, but keep in mind that you should always avoid including images to your bundle otherwise it will cause your bundle to be bulky and large which will lead to site startup delay, because your bundle is the first thing user download when it hits your website

Reply:
So I should write Img component like previous project?